tests/serialization.nim

```diff
@@ -13,20 +13,20 @@ import ../blscurve, std/strutils
 # A signature may be initialized at an infinity point
 # as a first step before aggregation. Inputs
 
-echo "\nZero init signatures is serialized as infinity point"
-echo "----------------------------------\n"
-
-proc test_zero_sig() =
+when BLS_BACKEND == BLST:
```

Comment:
will look into how to have MIRACL default init signature be serialized as an infinity point as well